astropy/timeseries/sampled.py

```diff
@@ -189,26 +192,36 @@ def fold(self, period=None, epoch_time=None, epoch_phase=0,
         period_sec = period.to_value(u.s)
 
         if normalize_phase:
+            if isinstance(epoch_phase, Quantity) and epoch_phase.unit.physical_type != 'dimensionless':
+                raise UnitsError('epoch_phase should be a dimensionless Quantity or a float when normalize_phase=True')
```

Comment:
these lines are super long, would you mind wrap earlier?

Reply:
Fixed